Scripts/Item Scripts/key.gd

```diff
@@ -0,0 +1,11 @@
+class_name Key extends Item
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready() -> void:
```

Comment:
pelase clean up unused functions

Reply:
yeah i missed the key class, the rest were cleaned, mb